src/Components/test/E2ETest/ServerExecutionTests/ServerSideAppTest.cs

```diff
@@ -56,13 +61,13 @@ public void NavMenuHighlightsCurrentLocation()
             Browser.FindElement(By.LinkText("Counter")).Click();
 
             // Verify we're now on the counter page, with that nav link (only) highlighted
-            WaitAssert.Equal("Counter", () => Browser.FindElement(mainHeaderSelector).Text);
+            Browser.Equal("Counter", () => Browser.FindElement(mainHeaderSelector).Text);
```

Comment:
What's the reason for changing these?

Reply:
They were relying on an async local and I wanted to avoid that as much as possible. I considered either passing the browser explicitly or making them extension methods, and I did the second because it felt more "idiomatic".

I didn't think a lot of one way or the other. Just wanted the async local usage limited to the minimum.

Reply:
But isn't this a big change in functionality? If these needed waitassert in the past, why is this safe to remove?

Reply:
I didn't remove anything. I transformed the method.
This used to be WaitAssert.Equal(string expected, Func<string> func) and internally used BrowserTestBase.Browser (an async local).

I converted it into WaitAssert.Equal(IWebDriver browser, string expected, Func<string> func) and to make it more similar to what it was I made WaitAssert.Equal and related methods extension methods on IWebDriver, so `WaitAssert.Equal(this IWebDriver browser, string expected, Func<string> func)`. 

Functionality wise they are equivalent. The use of an async local was unnecessary, when we can just pass the browser parameter explicitly.